web/gds-user-ui/src/modules/dashboard/member/utils.ts

```diff
@@ -87,3 +87,18 @@ export async function copyToClipboard(data: any) {
       console.error('[copyToClipboard]', err);
   }
 }
+
+export const getBusinessCategory = (category: any) => {
```

Comment:
This is a temporary fix for now because the BE is returning a number instead of a string. Here's a link to how the Business Categories are listed on the BE: https://github.com/trisacrypto/trisa/blob/0b895ae982a8162b729f67e54b36e619ef39b8bf/pkg/trisa/gds/models/v1beta1/models.pb.go#L28C1-L28C1

Once a fix is made to return strings, this will be removed. Also, I added the strings following the discussion during QA.